src/templates/addons-page/addons-page.stories.tsx

```diff
@@ -0,0 +1,430 @@
+	description: string;
+	icon: React.ReactNode;
+	installed?: boolean;
+}
+
+export default {
+	title: 'Templates/Addons/Addons Page',
+	parameters: {
+		layout: 'fullscreen',
+		a11y: {
+			config: {
+				rules: [
+					{
+						id: 'color-contrast',
+						enabled: true,
+					},
+					{
+						id: 'keyboard-navigation',
+						enabled: true,
+					},
+				],
+			},
+		},
+	},
+	decorators: [
+		( Story: React.ComponentType ) => (
+			<div
+				id="story-root-tw"
+				className="box-border [&_*]:box-border w-full overflow-x-hidden"
+			>
+				<Story />
+			</div>
+		),
+	],
+	tags: [ 'autodocs' ],
+};
+
+const addonsData: Addon[] = [
+	{
+		id: '1',
+		name: 'SureForms',
+		description:
+			'The new way to sell on WordPress. Create beautiful checkout experiences.',
+		installed: false,
+		icon: <SureFormsLogo className="size-8" />,
+	},
+	{
+		id: '2',
+		name: 'Spectra',
+		description:
+			'Supercharge your website with 40+ advanced blocks and templates.',
+		installed: true,
+		icon: <SpectraLogo className="size-8" />,
+	},
+	{
+		id: '3',
+		name: 'SureDash',
+		description:
+			'Advanced Gutenberg blocks for creating stunning websites effortlessly.',
+		installed: false,
+		icon: <SureDashLogo className="size-8" />,
+	},
+	{
+		id: '4',
+		name: 'Cartflows',
+		description:
+			'Comprehensive analytics and reporting for your WordPress site.',
+		installed: false,
+		icon: <CartFlowsLogo className="size-8" />,
+	},
+	{
+		id: '5',
+		name: 'SureCart',
+		description:
+			'Protect your website from threats with advanced security features.',
+		installed: false,
+		icon: <SureCartLogo className="size-8" />,
+	},
+	{
+		id: '6',
+		name: 'Ottokit',
+		description:
+			'Multi-language and translation support for global audiences.',
+		installed: true,
+		icon: <OttokitLogo className="size-8 text-support-info" />,
+	},
+	{
+		id: '7',
+		name: 'StarterTemplate',
+		description: 'Advanced user management and role customization tools.',
+		installed: false,
+		icon: <StartersTemplatesLogo className="size-8" />,
+	},
+	{
+		id: '8',
+		name: 'Presto Player',
+		description:
+			'Enhanced media management with advanced organization features.',
+		installed: false,
+		icon: <PrestoPlayerLogo className="size-8 text-brand-primary-600" />,
+	},
+	{
+		id: '9',
+		name: 'SureFeedback',
+		description:
+			'Boost your website speed with advanced caching and optimization.',
+		installed: true,
+		icon: <SureFeedbackLogo className="size-8 text-support-warning" />,
+	},
+];
+
+const themesData: Addon[] = [
+	{
+		id: '10',
+		name: 'Astra Theme',
+		description:
+			'Fast, lightweight, and customizable WordPress theme perfect for any website.',
+		installed: false,
+		icon: <AstraThemeLogo className="size-8 text-support-error" />,
+	},
+];
+
+interface AddonCardProps {
+	addon: Addon;
+	onInstall?: ( addon: Addon ) => void;
+	onConfigure?: ( addon: Addon ) => void;
+}
+
+const AddonCard = ( { addon, onInstall, onConfigure }: AddonCardProps ) => (
```

Comment:
Instead of adding the card as a separate component, write the card JSX directly into the main template. Otherwise component code will not be visible in the show code option.